spacy_llm/tests/tasks/test_entity_linker.py

```diff
@@ -0,0 +1,631 @@
+# ruff: noqa: W291
+import csv
+import functools
+from pathlib import Path
+
+import numpy
+import pytest
+import spacy
+from confection import Config
+from spacy import Vocab
+from spacy.kb import InMemoryLookupKB
+from spacy.tokens import Span
+from spacy.training import Example
+from spacy.util import make_tempdir
+
+from spacy_llm.registry import fewshot_reader, file_reader
+from spacy_llm.tasks.entity_linker import EntityLinkerTask, make_entitylinker_task
+from spacy_llm.tasks.entity_linker.util import SpaCyPipelineCandidateSelector
+from spacy_llm.util import assemble_from_config
+
+from ..compat import has_openai_key
+
+EXAMPLES_DIR = Path(__file__).parent / "examples"
+TEMPLATES_DIR = Path(__file__).parent / "templates"
+
+
+@functools.lru_cache()
+def build_el_pipeline(nlp_path: Path, desc_path: Path) -> None:
```

Comment:
We could use this function / toy use-case as an example for the `usage_examples` dir + one of the example files you already created for the tests 🙂

Reply:
Done in https://github.com/explosion/spacy-llm/pull/241/commits/21b27474eb905287930c5574d05ac199e9e53456.